src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerUdpBatcher.cs

```diff
@@ -31,8 +31,8 @@ public class JaegerUdpBatcher : IJaegerUdpBatcher
 
         private readonly SemaphoreSlim flushLock = new SemaphoreSlim(1);
         private readonly TimeSpan maxFlushInterval;
-        private readonly System.Timers.Timer maxFlushIntervalTimer;
-
+        private readonly System.Timers.Timer maxFlushIntervalTimer;
```

Comment:
It's not really possible to review your changes without ignoring whitespace changes.

Could you please revert these and leave only the relevant part for review?